api/authentication/tests.py

```diff
@@ -1 +1,69 @@
-# Create your tests here.
+import os
+
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from users.tests.mock import COMMON_USER, TestUserSet
+
+
+class CommonUserTestSet(TestCase):
+
+  @classmethod
+  def setUpTestData(cls):
+    cls.user = TestUserSet(**COMMON_USER)
+
+  def tearDown(self):
+    self.user.object.delete()
+
+
+class VariousEnvTokenResponseTest(CommonUserTestSet):
+
+  @classmethod
+  def setUpTestData(cls):
+    super().setUpTestData()
+    cls.credentials = {
+      "email": cls.user.object.email,
+      "password": "1234",
+    }
+
+  def test_token_response_attrs(self):
+    check_list_token_attrs = ["domain", "httponly", "secure"]
+    os.environ["DJANGO_SETTINGS_MODULE"] = "local"
+    response = self.user.client.post(
+      path=reverse("token_obtain_pair"),
+      data=self.credentials,
+    )
+    # token_detail transforms attributes of Morse object(http.cookie) into dict.
+    # It is contains cookies settings such as httponly, samesite.
+    token_detail = {key: value for key, value in response.cookies["refresh"].items()}
+    self.assertFalse(
+      any([token_detail[key] for key in check_list_token_attrs])
+    )
+    os.environ["DJANGO_SETTINGS_MODULE"] = "prod"
+    response = self.user.client.post(
+      path=reverse("token_obtain_pair"),
+      data=self.credentials,
+    )
+    token_detail = {key: value for key, value in response.cookies["refresh"].items()}
+    self.assertTrue(
+      all([token_detail[key] for key in check_list_token_attrs])
+    )
+
+
+class TokenRefreshViewTest(CommonUserTestSet):
+
```

Comment:
```suggestion
class TokenRefreshViewTest(CommonUserTestSet):

  @classmethod
  def setUpTestData(cls):
    super().setUpTestData()
    cls.refresh_token = RefreshToken().for_user(cls.user)
```
TokenRefreshViewTest에서 사용하는 refresh_token을 설정.